en/editor/assets/index.md

```diff
@@ -111,6 +111,28 @@ The search function is a combined function that can limit the search type and sp
 - Shortcut key `Esc` to cancel rename
 - In addition, the initial name of the Typescript asset will be treated as its `className`, and the `className` cannot be repeated.
 
+### Exporting Assets Package
+
+Select the resource to be exported and right click on it, then select **Export Resource Package**. The **Export Resource** panel will automatically list the currently selected resource and its dependencies. If you don't need to export dependent resources, you can uncheck **Include Dependent Resources** in the bottom left corner of the **Export Resources** panel.
+
+Once you have identified the resources to be exported, click the **Export** button to bring up the file storage dialog box where you will need to specify a folder location and file name.
```

Comment:
Once you have identified the resources to be exported, click the **Export** button to bring up the file storage dialog box where you will need to specify a folder location and file name.
-->
Once identified, the resources can be exported by clicking the **Export** button to bring up the file storage dialog box where it is necessary to specify a folder location and file name.